messaging/src/android/java/com/google/firebase/messaging/cpp/RegistrationIntentService.java

```diff
@@ -28,7 +27,7 @@
  * A class that manages Registration Token generation and passes the generated tokens to the native
  * OnTokenReceived function.
  */
-public class RegistrationIntentService extends IntentService {
+public class RegistrationIntentService extends JobIntentService {
```

Comment:
The CI tests are failing and I think it might be becuase you need to import the JobInentService here.

Reply:
Fixed